db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "TODO"
```

Comment:
This is used (exclusively I think) by metrics rollups for EmsClusters [ref](https://github.com/ManageIQ/manageiq/blob/master/app/models/metric/capture.rb#L170).  Identifier is `"#{parent.class}:#{parent.id}"` [ref](https://github.com/ManageIQ/manageiq/blob/master/app/models/metric/capture.rb#L152)

Reply:
Thanks, updated.